content/articles/genie-slide-in-jelly-and-scale-pop-up-notification-effects/index.md

```diff
@@ -0,0 +1,1107 @@
+  backface-visibility: hidden;
+}
+
+.ns-close:hover, .ns-close:focus {
+  outline: none;
+}
+
+/* Notification 'on close' before and after styling */
+.ns-close::after, .ns-close::before {
+  left: 50%;
+  top: 50%;
+  height: 60%;
+  width: 3px;
+  content: '';
+  background: #6e6e6e;
+  position: absolute;
+}
+
+.ns-close:hover::after, .ns-close:hover::before {
+  background: var(--white);
+}
+
+.ns-close::before {
+  transform: translate(-50%, -50%) rotate(45deg);
+  -webkit-transform: translate(-50%, -50%) rotate(45deg);
+}
+
+.ns-close::after {
+  transform: translate(-50%, -50%) rotate(-45deg);
+  -webkit-transform: translate(-50%, -50%) rotate(-45deg);
+}
+```
+
+The results for this styling is as shown in the image below:
+
+![add default notification box styling](engineering-education/genie-slide-in-jelly-and-scale-pop-up-notification-effects/add-default-notification-box-styling.png "add default notification box styling")
+
+Notice that the notification now looks more like a pop-up notification. The only issue is that it doesn't close
+automatically, neither can it be closed. Solve these problems by adding some CSS styling in
+the `growl-notification-style.css` file.
+
+### Add the growl effect to the notification
+In the `growl-notification-style.css` file, add growl effects. Find the code and some inline commentary of the growl effects CSS code [here](https://github.com/prograte/Genie-Slide-in-Jelly-and-scale-Pop-up-Notification-effects/blob/main/css/growl-notification-style.css).
+
+- Added growl styling that will be shared with all the types of growl effects
+- Added individual effects for each of the growl effect sets. This starts with the scale effect. The _scale_ effect copies its name. It expands from a small size to a larger size. It produces a notification with the animation shown in
+  the GIF below:
+
+![Scale effect gif](engineering-education/genie-slide-in-jelly-and-scale-pop-up-notification-effects/scale-effect-gif.gif "Scale effect gif")
+
+- Added the _Jelly_ animation effect. Some part of it is generated from [Bouncejs.com](http://bouncejs.com/).
+
+  > [Bouncejs.com](http://bouncejs.com/) is a site used to generate awesome CSS3 animations for a website. Click on the site to check out some presets available or create animations that are tailored for the site.
+
+Head over to the `growl-effects-jelly.html` either by opening it in a browser or clicking the `Jelly` button on the
+webpage. Click on `SHOW NOTIFICATION` to see the animation. The animation looks like that in the GIF below:
+
+![Jelly effect gif](engineering-education/genie-slide-in-jelly-and-scale-pop-up-notification-effects/jelly-effect-gif.gif "Jelly effect gif")
+
+- Added the slide-in animation to the project. Open the `growl-effects-slide-in.html` by clicking on the `Slide in`
+  button. This file has the notification style set to slide hence it will implement the `slide in` effect. The effect is
+  displayed in the GIF shown below:
```

Comment:
```suggestion
Added the slide-in animation to the project. Open the `growl-effects-slide-in.html` by clicking on the `Slide in` button. This file has the notification style set to slide hence it will implement the `slide in` effect. The effect is displayed in the GIF shown below:
```